spec/factories.rb

```diff
@@ -1,4 +1,19 @@
 FactoryBot.define do
+  factory :order do
+    cart { nil }
```

Comment:
If you have associations defined, you could use `association :user`. You could take a look here for more details: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations